internal/connectors/plugins/public/adyen/plugin.go

```diff
@@ -16,6 +18,8 @@ type Plugin struct {
 }
 
 func (p *Plugin) Install(ctx context.Context, req models.InstallRequest) (models.InstallResponse, error) {
+	ctx, span := otel.StartSpan(ctx, "adyen.Install")
```

Comment:
I am wondering if we can put that outside the plugin implementation, but in the grpc implementation (since we can add which plugin it is, we can have it directly in that file: https://github.com/formancehq/payments/blob/feat/payments-v3/internal/connectors/plugins/grpc_implementation.go#L24) What do you think ?

That way, if a client adds a connector in the future, they will not have to think about adding a span everywhere in their code

Reply:
Good idea. I've reimplemented it so that we send the psp name as an attribute.